ui/components/multichain/pages/connections/connections.stories.js

```diff
@@ -0,0 +1,19 @@
+import React from 'react';
+import { Connections } from './connections';
+
+export default {
+  title: 'Components/Multichain/Connections',
+};
+export const DefaultStory = () => (
+  <div
+    style={{
+      width: '400px',
+      height: '600px',
+      border: '1px solid red',
```

Comment:
Question: can we remove the styles of the higher dom? it seems like most of stories in storybook didn't have this one.

Reply:
I think we can take out the border style, but I defined the width and height of the container to make sure the flex alignments were visually correct